scripts/backend_test_shards.json

```diff
@@ -154,7 +154,12 @@
         "core.platform.email.dev_mode_email_services_test",
         "jobs.blog_validation_errors_test",
         "jobs.blog_validation_jobs_test",
-        "jobs.transforms.blog_validation_test"
+        "jobs.transforms.blog_validation_test",
+        "core.tests.build_sources.extensions.base_test",
+        "core.tests.build_sources.extensions.models_test",
+        "core.tests.load_tests.feedback_thread_summaries_test",
```

Comment:
There is a specific flage (`--exclude_load_tests`) for this file in `run_backend_tests.py`, you might want to check if we want to add this here.

Reply:
@vojtechjelinek I think we should be running all tests on CI, so we should include this file. Note that if a shard is specified, `--exclude_load_tests` does not affect which tests run

Reply:
Waiting for this to be resolved, either confirm that the test is not flaky or remove it from here, as we discussed offline.

Reply:
I have confirmed that the test is not flaky

Reply:
One more thing (that can be done in a different PR) should we move the test in some feedback related test file?